packages/react-tinacms-inline/src/blocks/inline-block-field-controls.tsx

```diff
@@ -261,6 +272,51 @@ export const BlockMenuWrapper = styled.div<BlockMenuWrapperProps>(p => {
   `
 })
 
+interface BlockLabelProps {
```

Comment:
Could we separate this into a new file?

Reply:
I guess since the other components are there, it makes sense. But it may be time to break this file up into smaller bits in a block controls directory, considering the size.

Reply:
For sure, we should refactor this soon.

Reply:
One of the changes I'd like to make soon is the ability to disable the directional controls and/or the add block buttons. I think that will be a more extensive change and maybe a good time to reorganize.